src/common/pubsub.ts

```diff
@@ -202,7 +202,7 @@ export const notifyView = (
   log: EventLogger,
   postId: string,
   userId: string,
-  referer: string,
+  referer: string | undefined,
```

Comment:
can't make it `referer?` because it is not object argument and don't want to refactor all over the place